vit/command_bar.py

```diff
@@ -7,6 +7,7 @@ class CommandBar(urwid.Edit):
     def __init__(self, **kwargs):
         self.event = kwargs['event']
         self.autocomplete = kwargs['autocomplete']
+        self.abort_backspace = kwargs.pop('abort_backspace')
```

Comment:
We're not using leftover ```kwargs```, so the ```pop()``` seems unnecessary or inconsistent here. We should either ```pop()``` all the ```kwargs``` or none of them.

Reply:
The `kwargs` is also used by the parent constructor. Without `pop()`, an exception would be raised:

>   File "command_bar.py", line 16, in __init__
>    return super().__init__(**kwargs)
> TypeError: __init__() got an unexpected keyword argument 'abort_backspace'

Reply:
Ah, yep, that makes sense. Maybe a code note in there to clarify why we're ```pop```ing it?

Reply:
Good idea. I'll do it.